src/surfaceComponent/Card/Card.tsx

```diff
@@ -0,0 +1,74 @@
+import { PropsWithChildren } from "react";
+import * as React from "react";
+import { Button } from "../../inputs";
+import smallImage from "../../static/images/card/card-small.png";
+import bigImage from "../../static/images/card/card-big.png";
+import * as styles from "./Card.module.css";
+
+interface CardProps {
+  fullWidth?: boolean;
+}
+
+export const Card = ({ fullWidth = false }: PropsWithChildren<CardProps>) => {
+  const handlePrimaryButton = () => {};
+  const handleSecondaryButton = () => {};
+
+  const imageUrl = fullWidth ? bigImage : smallImage;
+
+  const getIfFullWidthCard = fullWidth ? styles.fullWidth : "";
+
+  const getIfFullWidhtCardContent = fullWidth ? styles.fullWidthCardContent : ""
+
+  const getIfFullWidthButtonWrapper = fullWidth ? styles.buttonWrapperFullWidth : "";
+
+  const cardDynamicClasses = [
+    styles.cardWrapper, 
+    getIfFullWidthCard
+  ].join(" ");
+
+  const cardContentDynamicClasses = [
+    styles.cardContent,
+    getIfFullWidhtCardContent,
+  ].join(" ")
+
+  const buttonWrapperDynamicClasses = [
+    styles.buttonWrapper, 
+    getIfFullWidthButtonWrapper,
+  ].join(" ")
+
+  return (
+    <div className={cardDynamicClasses}>
+      <img src={imageUrl} alt="" />
+      <div className={cardContentDynamicClasses}>
```

Comment:
**issue:** What if I want to add third Button here? Or change paragraph description? You hardcoded everything in this component - it is impossible to re-use it for other purposes. Check desired implementation in your task specification for more details on how should it look like.